HelpSource/Classes/AbstractFunction.schelp

```diff
@@ -121,7 +121,33 @@ a = Pseries(220, 220, inf).cpsmidi; a.asStream.nextN(12); // overtone series as
 method::midiratio
 method::ratiomidi
 method::ampdb
+Converts amp range (1.0 -> 0.0) to db range (ex. 0dB -> -90dB).
+code::
+(
+Window.closeAll;
```

Comment:
can you please remove this line in both examples? it's not necessary and creates a risk that users might close something important. it's also not consistent with other examples throughout our documentation.